src/BTConnectionManager.cpp

```diff
@@ -96,23 +95,35 @@ BTConnectionManager::BTConnectionManager(QObject* parent)
         d->discoveryRunning = false;
         emit discoveryRunningChanged(d->discoveryRunning);
     });
+
```

Comment:
All of these unrelated changes are extraordinarily distracting when doing a review, cleanup is appreciated but please submit those changes as a separate pull request in the future.

Reply:
Ok, I did those small changes in order to understand what is going on at the code. In the future I will try to make them at a separated pull request, you are right.

Reply:
Thank you :)